tests/assetfetcher_test.go

```diff
@@ -0,0 +1,276 @@
+// +build integration
+
+package inttest
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	packr "github.com/gobuffalo/packr/v2"
+	pq "github.com/lib/pq"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/asecurityteam/ipam-facade/pkg/assetfetcher"
+	"github.com/asecurityteam/ipam-facade/pkg/domain"
+	"github.com/asecurityteam/ipam-facade/pkg/sqldb"
+	"github.com/asecurityteam/settings"
+)
+
+var ctx context.Context
+var conn *sql.DB
+var db *sqldb.PostgresDB
```

Comment:
suggestion: I know these are integration tests, but I'd feel more confident if we could do this without sharing as much state. At the very least, I'd like to see each test run in its own schema instance rather than sharing one.

Ideally, all of our tests could be run in parallel. As it is, these tests would start to fail if we enabled parallel test runs due to sharing a single schema and dropping all data as they begin.